content/en/apps/guides/hosting/monitoring/setup.md

```diff
@@ -0,0 +1,168 @@
+
+[Prometheus supports](https://prometheus.io/docs/concepts/metric_types/) four metric types: Counter, Gauge, Histogram, and Summary.  Currently, the CHT only provides Counter and Gauge type metrics. When building panels for Grafana dashboards, [Prometheus Functions](https://prometheus.io/docs/prometheus/latest/querying/functions/) can be used to manipulate the metric data. Refer to the [Grafana Documentation](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/best-practices/) for best practices on building dashboards.
+
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- URL(s) of the CHT instance(s)
+
+### Setup
+
+These instructions have been tested against Ubuntu, but should work against any OS that meets the prerequisites. They follow a happy path assuming you need to only set a secure password and specify the URL(s) to monitor:
+
+1. Run the following commands to clone this repository, initialize your `.env` file, create a secure password and create your data directories:
+
+    ```sh
+    cd ~
+    git clone https://github.com/medic/cht-monitoring.git
+    cd cht-monitoring
+    cp cht-instances.example.yml cht-instances.yml
+    cp grafana/grafana.example.ini grafana/grafana.ini
+    mkdir -p grafana/data && mkdir  -p prometheus/data 
+    apt install -y wamerican  # ensures /usr/share/dict/words is present for shuf call below 
+    cp .env.example .env
+    password=$(shuf -n7 /usr/share/dict/words --random-source=/dev/random | tr '\n' '-' | tr -d "'" | cut -d'-' -f1,2,3,4,5,6,7)
+    sed -i -e "s/password/$password/g" .env
+    echo;echo "Initial project structure created! To log into Grafana in the browser:";echo 
+    echo "    username: medic"
+    echo "    password: ${password}";echo
+    ```
+   Note that in step 4 below you'll need the username and password which is printed after you run the above command.
+
+2. Edit the `cht-instances.yml` file to have the URLs of your CHT instances. You may include as many URLs of CHT instances as you like. To preserve data consistency parameters in this file should NOT be adjusted after it is initially set.
+
+   Here is an example:
+
+    ```yml
+    - targets:
+      - https://subsub.sub.example.com
+      - https://cht.domain.com
+      - https://website.org
+    ```
+3. Run the following command to deploy the stack:
+
+    ```sh
+    cd ~/cht-monitoring
+    docker compose up -d
+    ```
+
+4. Grafana is available at [http://localhost:3000](http://localhost:3000). See the output from step 1 for your username and password.
+
+If you would like to do more customizing of your deployment, see ["Addition Configuration"](#additional-configuration).
+
+### Upgrading
+
+Before upgrading, you should back up both your current configuration settings and your Prometheus time-series data.
```

Comment:
```suggestion
Before upgrading, you should back up both your current configuration settings as well as your Prometheus/Grafana data directories.
```

The Grafana data directory should also be backed up since it might contain custom dashboards/configuration/etc.